src/frontend/src/lib/components/tokens/TokenLogo.svelte

```diff
@@ -42,6 +46,7 @@
 				src={badge.blackAndWhite ? networkIconBW : networkIcon}
 				alt={replacePlaceholders($i18n.core.alt.logo, { $name: networkName })}
 				{color}
+				testId={badgeTestId}
```

Comment:
Is it expected that you use twice the same `testId` within the component?

Reply:
the component will render one or the other badge, it is exclusive; shall i maybe add a suffix/prefix, line `token-count-{badgeTestId}` and `network-{badgeTestId}` ?

Reply:
Maybe? Somehow always thought `testId` are kind of unique.

Reply:
done! not really sure if should be prefix or suffix; have a look, WDYT? any preference?